packages/sdk-core/package.json

```diff
@@ -9,18 +9,27 @@
     "module": "dist/module/index.js",
     "types": "dist/main/index.d.ts",
     "files": [
-        "dist/**/*.js",
-        "dist/**/*.d.ts"
+        "dist/main",
+        "dist/module",
+        "dist/docs",
+        "dist/index.umd.js",
+        "!**/*.json",
+        "CHANGELOG.md",
+        "LICENSE",
+        "README.md"
     ],
     "scripts": {
-        "build": "yarn generate-abi-files && yarn generate-web3-types && yarn generate-graphql-types && rm -rf dist && rollup --config rollup.config.ts --configPlugin typescript",
+        "build": "yarn generate-abi-files && yarn generate-web3-types && yarn generate-graphql-types && rm -rf dist && run-p build:*",
+        "build:main": "tsc -p tsconfig.json",
+        "build:module": "tsc -p tsconfig.module.json",
+        "build:umd": "browserify src/index.ts -s sdkCore -p [ tsify --noImplicitAny ] > dist/index.umd.js",
```

Comment:
Browserify does the UMD magic here by creating a global variable names `sdkCore`. Feel free to change the name.

Reply:
name is good